_jobs/software-engineer.md

```diff
@@ -10,52 +10,61 @@ Compiler is a small, woman-owned software consultancy. Compiler helps service-or
+- Self-direct and time-manage across multiple client and internal projects with varying levels of complexity, demand, and constraint
 
-## Preferred qualifications
+## Preferred Qualifications
 
 - Experience working with or in proximity to government
 - Experience using modern, high-level programming languages such as Python and JavaScript to build web applications, API services, or similar
+- Experience developing performant and accessible front-end components that meet WCAG standards, using advanced HTML and CSS
 - Experience using Docker and containers to manage reproducible environments
-- Experience deploying production applications and services to cloud environments such as AWS and GCP
-- Experience collaborating with product managers and remote teams
-- Knowledge of Git / GitHub based development workflows and open source project management
+- Experience deploying and managing production applications and services in cloud environments such as Azure and GCP, preferably with Infrastructure as Code systems such as Terraform
+- Experience collaborating with product managers, UI/UX designers, remote teams, along with client technical teams
+- Knowledge of Git / GitHub-based development workflows and open source project management
 - A love of public transportation
-- A personal and professional commitment to bring anti-racist principles into development practices
+- A personal and professional commitment to bring anti-racist principles into your work
+- Preference given to candidates located in California, and specifically Los Angeles
 
-## Position type
+## Education
 
-Full-time (~30 hours/week) W2 employment
+- Per company policy, this position has no formal educational requirements
 
-## Start date
+## Start Date
 
-Approximately Tuesday, August 3rd, 2021
+- Approximately Monday, April 1st, 2024
 
 ## Compensation
 
-Expected salary range is $100,000 - $120,000 annually based on qualifications and experience
-Paid time off: 15 days paid time off (PTO) and 10 sick days annually, federal holidays + Juneteenth, and holiday closure between December 23 and January 1
-Medical, dental, and vision insurance benefits
-
-## Location
-
-100% remote; the current team is spread across Pacific and Eastern time zones in the US.
+- Expected salary range is $100,000 - $120,000 annually based on qualifications and experience
+- Paid time off: fifteen (15) days paid time off (PTO) and ten (10) sick days annually, federal holidays, and annual holiday closure between December 23 and January 1
+- 100% Company-paid medical, dental, and vision insurance premiums for employees; 50% Company-paid insurance premiums for employees’ dependents
+- 100% Company-paid short- and long-term disability insurance
+- Discretionary budget for home office and/or professional development
+- Company 401k contribution
+- Company-paid in-person retreats twice a year
```

Comment:
One paid (required?) conference of your choice

Reply:
Thanks for this suggestion too @machikoyasuda. We’re trying to turn this into a formal policy but we feel we don’t want to share it with potential candidates as a perk until it’s formalized, as part of our budget process for 2024.